Tests/MongoSwiftSyncTests/SpecTestRunner/TestOperation.swift

```diff
@@ -470,126 +619,150 @@ extension WriteModel: Decodable {
 }
 
 struct BulkWrite: TestOperation {
+    let session: String?
     let requests: [WriteModel<Document>]
     let options: BulkWriteOptions
 
-    func execute(on target: TestOperationTarget, session: ClientSession?) throws -> TestOperationResult? {
+    private enum CodingKeys: CodingKey { case session, requests }
+
+    init(from decoder: Decoder) throws {
```

Comment:
my comment for `InsertMany` applies here as well

Reply:
Done.